### ISSUE TITLE
awaiting-review added to PRs that already have the label

### ISSUE THREAD

**Issue by finnhodgkin** [dwyl/dwylbot#165]
Sorry if this has already been raised but dwylbot has been doing this to every one of my PRs: 

![selection_041](https://user-images.githubusercontent.com/22300773/30818765-66ac8b76-a214-11e7-828b-92e820cb5f28.png)

**Comment by SimonLab** [comment]
Yes I know it annoys everyone this bug :) I think there is already an issue for this bug.
I think also a PR which was supposed to fix this has been merged so I'll check if the application has been deployed otherwise I suspect there is still a bug. I need to fix this asap, thanks for the reminder @finnhodgkin

**Comment by finnhodgkin** [comment]
Haha sorry! I did do a search before raising, though didn't check closed issues... :smile:

**Comment by ghost** [comment]
I wouldn't say this needs to be fixed asap as it doesn't actually cause any problems aside from a visual annoyance - it just reapplies a label that already exists, should definitely be fixed at some point though
